### ISSUE TITLE
importing anything in my gambit build fails

### ISSUE THREAD

**Issue by amirouche** [gambit/gambit#424]
It seems to me there is a problem with gambit master, doing `(import (scheme base))` at the REPL leads to my laptop freezing:

```scheme
$ cd submodules/gambit/
amirouche@ubujan19:submodules/gambit$ git show | head -n 1
commit 36b474a6a33b49ae3dc2dee84568ac57d26acec1
amirouche@ubujan19:submodules/gambit$ timeout 15 ./bin/gsi-script 
 Gambit v4.9.3 

> (import (scheme base))            
amirouche@ubujan19:submodules/gambit$ echo $?
124
```

**Comment by amirouche** [comment]
Actually, it is not just `scheme base` but anything.

**Comment by lassik** [comment]
What's the output of `./bin/gsi-script -v`?

**Comment by amirouche** [comment]
$ ./bin/gsi-script -v
v4.9.3 20180930122740 x86_64-pc-linux-gnu "./configure"

**Comment by amirouche** [comment]
$ git show | head -n 1
commit 36b474a6a33b49ae3dc2dee84568ac57d26acec1

**Comment by feeley** [comment]
The problem is that you did not `make install`.  The runtime system is expecting `lib` to be in the Gambit installation directory (what is known as `~~`), and it is not there.  You can force a specific directory with:

```
gsi -:=~~=<dir>
```

so you should do

```
./bin/gsi-script -:=.
```

**Comment by feeley** [comment]
The infinite loop bug is fixed by commit 509d906265934db309f85fb0bd1a6dd7015c2322 (this was caused by `import`'s expander being defined in module `_define-library` which is unavailable when the Gambit installation directory isn't set properly... now it gives an error).

**Comment by amirouche** [comment]
Thanks!
